cmd/traffic_layout/traffic_layout.cc

```diff
@@ -251,8 +252,7 @@ main(int argc, const char **argv)
 {
   // check for traffic_runroot operations
   for (int i = 0; i < argc; i++) {
-    if (!strcmp(argv[i], "--init") || !strcmp(argv[i], "--remove") || !strcmp(argv[i], "-i") || !strcmp(argv[i], "-r") ||
-        !strcmp(argv[i], "--force")) {
+    if (argv[i] == "init"_sv || argv[i] == "remove"_sv || argv[i] == "--force"_sv || argv[i] == "--relocate"_sv) {
```

Comment:
So this is doing subcommands now? In that case, I'd recommend you look at `traffic_ctl` for how to do subcommands.